projects/optic/ci/configs/github.yml

```diff
@@ -23,8 +27,12 @@ jobs:
           # If true, standard check failures will cause this action to fail.
           # If false, standard check failures will show in PR comments and
           # in Optic Cloud but will not cause the action to fail
-          standards_fail: %standards_fail
+          standards_fail: true
 
           # If you have more than one spec, separate matches with commas
-          # (openapi.yml,other.yml)
-          additional_args: --match %match
+          # Examples of match: "openapi.yml" or "openapi.yml,other.yml" or "*.spec.yml"
+          additional_args: --match ""
```

Comment:
maybe the default here could be `openapi.yml`?  Either way, what happens if the user doesn't change this and runs diff-all in ci? Do we silently fail or pass? I can't remember how this works off the top of my head but I think this should fail so the user knows there's something to fix here

Reply:
This will match any spec by default, which I think is a great default. When no api is found, diff-all throws